CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 The change log describes what is "Added", "Removed", "Changed" or "Fixed" between each release.
 
+## Unreleased
+
+### Changed
+
+- The `Http\HttplugBundle\Collector\Collector` `getClientStacks` method have been renamed to `getClientRootStacks`
+and now returns only stacks without any parent.
```

Comment:
i am fine to remove this if you want.

Reply:
Nice, I didn't saw the _without_ in you previous comment!